code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -159,11 +159,13 @@
 
 	return ..()
 
-/mob/living/carbon/human/grabbedby(mob/living/carbon/user, supress_message = 0)
-	if(user == src && pulling && !pulling.anchored && grab_state >= GRAB_AGGRESSIVE && (disabilities & FAT) && ismonkey(pulling))
-		devour_mob(pulling)
-	else
-		..()
+// BROKEN
+// This supposed to be the code for devouring monkeys but it became broken
+// /mob/living/carbon/human/grabbedby(mob/living/carbon/user, supress_message = 0)
+// 	if(user == src && pulling && !pulling.anchored && grab_state >= GRAB_AGGRESSIVE && (disabilities & FAT) && ismonkey(pulling))
+// 		devour_mob(pulling)
+// 	else
+// 		..()

```

Comment:
dont comment out code like this and also this isnt really broken this is a fake fix